lib/user.py

```diff
@@ -8,9 +8,11 @@
 import base64
 import uuid
 import re
+from lib.document import Document
 from collections import defaultdict
 from responseprinter import ResponsePrinter
 
+PRINTER = ResponsePrinter()
```

Comment:
I was just emphasizing that it is global and will be shared among threads but if that is the convention for globals, I'm in agreement.